### PR TITLE
Make /getubuntu redirects more greedy

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Make /getubuntu redirects more greedy
- Looking at 404's people are also going to sub-pages of getubuntu

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/getubuntu/download
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are sent to /download

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -460,8 +460,8 @@ foundation-cloud-build/?: "/openstack/consulting"
 frs/?: "/"
 gateways/?:  "/internet-of-things/gateways"
 get/?: "/download"
-getubuntu/?: "/download"
-GetUbuntu/?: "/download"
+getubuntu.*/?: "/download"
```
Comment (tbille) [inline redirects.yaml:463]: it seems we are still getting a bit of  traffic on pages that don't 404

eg. ` /getubuntu/releasenotes? -> https://wiki.ubuntu.com/Releases`

Comment (anthonydillon) [reply redirects.yaml:463]: Can you redirect selected URLs from the 404s to save redirecting URLs that should 404 like /getubuntu/404 and not redirect existing paths.

Comment (carkod) [reply redirects.yaml:463]: These comments seem to be addressed, it's redirecting all to `/download`

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**